tests/manage/pv_services/test_pvc_rwx_writeable_after_pod_deletions.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+class TestRWXMountPoint(ManageTest):
+    """
+    Automates the following test cases:
+    OCS-965	CEPHFS RWX: While two app pods access same mount,
+        delete one pod. Confirm second pod is still able to write
+    """
+
+    @tier1
+    @pytest.mark.parametrize(
```

Comment:
if you don't plan to add also CephFilesystem interface, please remove parametrize

Reply:
@RazTamir done. Thanks

Earlier we thought we need to cover for RBD RWX as well. But we don't need to test RBD + RWX with all scenarios as its a special usecase. Hence removed the parameterize as suggected by you.

If in future we want to include RBD, we can again add the parameterize.